albumentations/augmentations/transforms.py

```diff
@@ -3107,3 +3108,46 @@ def apply(self, img, transforms=(), **params):
 
     def get_transform_init_args_names(self):
         return ("brightness", "contrast", "saturation", "hue")
+
+
+class Emboss(ImageOnlyTransform):
+    """Emboss the input image and overlays the result with the original image.
+
+    Args:
+        alpha ((float, float)): range to choose the visibility of the embossed image. At 0, only the original image is
+            visible,at 1.0 only its embossed version is visible. Default: (0.2, 0.5).
+        strength ((float, float)): strength range of the embossing. Default: (0.2, 0.7).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+    """
+
+    def __init__(self, alpha=(0.2, 0.5), strength=(0.2, 0.7), always_apply=False, p=0.5):
+        super(Emboss, self).__init__(always_apply, p)
+        self.alpha = to_tuple(alpha, 0.0)
+        self.strength = to_tuple(strength, 0.0)
+        self._emboss_matrix = self._generate_emboss_matrix()
+
+    def _generate_emboss_matrix(self):
+        alpha_sample = random.uniform(*self.alpha)
+        if not 0 <= alpha_sample <= 1.0:
+            raise ValueError(f"Expected 'alpha' to be in the interval [0.0, 1.0], got {alpha_sample}")
```

Comment:
I think, that better to check `alpha` range after `to_tuple` method, when we save it's range

Reply:
I think _emboss_matrix  should be generated for each new call, rather than being re-used. @KiriLev @Dipet

Reply:
made same as in Sharpen